proc_square_refl.py

```diff
@@ -18,7 +18,7 @@
         #('181103','probe',True),
```

Comment:
works with a524dcad0c2c33b3e7c93a81bf405d14e782928f but not current guess_function branch of pyspecdata. produces error:
`Traceback (most recent call last):
  File "proc_square_refl.py", line 150, in <module>
    d = d['t':(-10e6,10e6)] # slice out frequencies with signal
  File "c:\users\alex guinness\gitstuff\pyspecdata\pyspecdata\core.py", line 6294, in __getitem__
    slicedict,axesdict,errordict,unitsdict = self._parse_slices(args)
  File "c:\users\alex guinness\gitstuff\pyspecdata\pyspecdata\core.py", line 6486, in _parse_slices
    logger.debug(strm("i found",thisaxis[temp_high]))
IndexError: index 12500 is out of bounds for axis 0 with size 12500`

Reply:
again, fixed pyspecdata bug